stac_pydantic/shared.py

```diff
@@ -1,9 +1,16 @@
-from datetime import datetime
+from datetime import timezone
 from enum import Enum, auto
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Annotated, Any, Dict, List, Optional, Tuple, Union
```

Comment:
you'll need to import `Annotated` from `typing_extensions` for compatibility with python 3.8